bin/pycbc_create_injections

```diff
@@ -24,14 +24,14 @@ The configuration file should have a [variable_args], a [static_args],
 and one or more [distribution] sections. Multiple [constraint] sections and
 one or more [waveform_transforms] sections may also be provided. An example:
 
-    [variable_args]
+    [variable_params]
```

Comment:
Looks like the docs in line 23 has to be updated too.

Reply:
Whoops, good catch.